cks/templates/configmap.yaml

```diff
@@ -1,19 +1,15 @@
----
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ include "cks.fullname" . }}
   labels:
     {{- include "cks.labels" . | nindent 4 }}
 data:
-  AUTH_TOKEN_STORAGE_IN_MEMORY_ENCODING: base64
-  AUTH_TOKEN_STORAGE_TYPE: in-memory
-  BOOT_CHECKER_NOKEYS_RULE: importPEM
-  LOG_LEVEL: DEBUG
-  NEW_RELIC_ENABLED: "false"
-  NEW_RELIC_NO_CONFIG_FILE: "true"
+  AUTH_TOKEN_STORAGE_TYPE: "in-memory"
+  LOG_RSYSLOG_ENABLED: "false"
+  LOG_CONSOLE_ENABLED: "true"
   PORT: "{{ .Values.deployment.port }}"
   BOOT_CHECKER_IMPORT_PUBLIC_KEY_PATH: "{{ .Values.virtruKeys.mountPath }}/{{ .Values.virtruKeys.data.publicKey }}"
   BOOT_CHECKER_IMPORT_PRIVATE_KEY_PATH: "{{ .Values.virtruKeys.mountPath }}/{{ .Values.virtruKeys.data.privateKey }}"
```

Comment:
won't virtruKeys.data here also cause issues?

Reply:
Yep. hardcoded for now, but I don't think this value even matters I commented it out in my CKS deployment and I can still decrypt no problem